storage/tests/test_mvcc.rs

```diff
@@ -34,3 +34,137 @@ This file should comprise a set of low-level tests relating to MVCC.
    After cleanup is run, if we iterate directly on the storage layer, we should be able to confirm the keys are actually not present anymore (Rocks may defer the disk delete till compaction, but to us they are "gone").
 
  */
+
```

Comment:
Comment #4 above is still pending.
If we're happy with the current case, I'll move it to a TODO comment.